app/helpers/form_helper.rb

```diff
@@ -0,0 +1,10 @@
+module FormHelper
+  def bool_to_int bool
```

Comment:
this method is no longer needed (see my other comment)
I would be opposed to monkey patching core ruby classes when there are other reasonable ways to do this.